Benchmarks/Package.swift

```diff
@@ -44,5 +44,17 @@ let package = Package(
                 .plugin(name: "BenchmarkPlugin", package: "package-benchmark")
             ]
         ),
+        .executableTarget(
+            name: "DataIOBenchmarks",
+            dependencies: [
+                .product(name: "FoundationEssentials", package: "swift-foundation"),
```

Comment:
I think you likely want `swift-foundation-local` here instead of `swift-foundation`

Reply:
Indeed, thanks,.